guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,200 @@
+   alt-text = "Diagram of three federations with user avatars. A user deposits bitcoin into Federation 1 and receives ecash. A gateway with a lightning symbol represents the bitcoin lightning network connecting the federations."
+   width = 800
+   height = 500
+   layout = "full-width"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/federated-mint-2.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/federated-mint-2@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/federated-mint-2-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/federated-mint-2-mobile@2x.jpg"
+   caption = "Ecash transfers within a federation are instant, free, and private."
+   modalImage = "/assets/images/guide/how-it-works/ecash/federated-mint-2@2x.jpg"
+   alt-text = "Illustration showing members of Federation 1, represented by user avatars and ecash icons within a blue circle. Dashed arrows connect the ecash icons, symbolizing instant, free, and private ecash transfers between members of the same federation. The text reads, ‘Members of the same federation can send ecash with each other instantly, for free, and privately.’"
+   width = 800
+   height = 500
+   layout = "full-width"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/federated-mint-3.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/federated-mint-3@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/federated-mint-3-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/federated-mint-3-mobile@2x.jpg"
+   caption = "Federations are controlled by guardians who manage the issuance and redemption of ecash and bitcoin."
+   modalImage = "/assets/images/guide/how-it-works/ecash/federated-mint-3@2x.jpg"
+   alt-text = "Diagram showing Federation 1 controlled by four guardians, each with a server and a set of keys. Federation 1 includes a user, an ecash icon, and a bitcoin icon connected by dashed lines."
+   width = 800
+   height = 500
+   layout = "full-width"
+%}
+
+Federated Mints are the standard when it comes to security and resilience in the Fedimint protocol. Managed by four or more guardians, each running their own Fedimint server, this setup distributes control and minimizes risk.
+
+#### Pros:
+
+* **Security** - No single guardian can control the mint, reducing the risk of theft or mismanagement.
+
+* **High resilience** - The mint remains operational even if one or more guardians are temporarily offline, ensuring continuous access.
+
+#### Cons:
+
+* **Complex setup** - Coordinating multiple guardians adds steps and cost.
+
+## Role of guardians
+Guardians are participants who collaboratively manage a federation's operations. They secure the federation's funds using multi-sig wallets, validate payments through consensus, and control ecash issuance and redemption.
+
+## Role of gateways
+
+{% include picture.html
```

Comment:
The usage of photos make it a bit harder to parse the diagram. Using the Bitcoin symbol to represent Fedimint ecash feels counter-intuitive.